projects/ngx-datatable/src/lib/components/body/progress-bar.component.ts

```diff
@@ -1,14 +1,29 @@
-import { Component, ChangeDetectionStrategy } from '@angular/core';
+import { Component, ChangeDetectionStrategy, Input } from '@angular/core';
```

Comment:
I am not sure if this component needs to be changed. I think either the default is used from the table, or a custom one. The the internal here should not need to know about the custom one.

Am I right?